Makefile

```diff
@@ -1,5 +1,7 @@
 all: install_requirements configure_host launch_mgmt_cluster verify
 
+ci_config: configure_host launch_mgmt_cluster verify
```

Comment:
I'm thinking maybe `run` or something similar would be easier to understand?

Reply:
or just "ci", If I phrase it in a sentence it makes sense "The job uses the CI Make target" .

Reply:
I just make it as ci_run